extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -52,6 +53,10 @@ export class AmpAd extends AMP.BaseElement {
         // Unspecified or empty type.  Nothing to do here except bail out.
         return null;
       }
+      const config = adConfig[type];
```

Comment:
right, but getting config is not part of the code we want to share because it's too straightforward. what do you think about setting config in each element's `#buildCallback`?
`layoutCallback` would be too late, because sticky-ad need this info before `#scheduleLayout`